hash/msx1_cart.xml

```diff
@@ -17903,7 +17916,8 @@ legacy FM implementations cannot find it.
 	</software>
 
 	<!-- TODO possibility to add a second drive -->
-	<software name="hb3600">
+	<!-- CP/M image not recognized. -->
+	<software name="hb3600" supported="partial">
```

Comment:
I am bit in dubio whether this should be marked marked as partially supported.
The floppy interface emulation works fine. It is just that one of the dumped floppies that came with the interface is not working/recognized.